arbnode/dataposter/data_poster.go

```diff
@@ -336,7 +336,7 @@ func (p *DataPoster) UsingNoOpStorage() bool {
 	return p.usingNoOpStorage
 }
 
-var ErrExceedsMaxMempoolSize = errors.New("posting this transaction will exceed max mempool size")
+var ErrExceedsMaxMempoolSize = errors.New("Node is behind, posting this transaction will exceed max mempool size")
```

Comment:
This isn't due to the node being behind. `confirmDataPosterIsReady` should prevent this from being reached, but clearly something's not quite working there. Ignoring this error when it's ephemeral is fine though.